layers/meta-balena-raspberrypi/recipes-core/images/resin-image.bbappend

```diff
@@ -20,6 +20,7 @@ RESIN_BOOT_PARTITION_FILES_rpi = " \
 
 RESIN_BOOT_PARTITION_FILES_append_raspberrypi4-64 = " \
     rpi-eeprom/pieeprom-latest-stable.bin:/pieeprom-latest-stable.bin \
+    rpi-eeprom/vl805-latest-stable.bin:/vl805-latest-stable.bin \
```

Comment:
Can we leave some info in the commit log about where this file was taken from?

Reply:
sure, done